superset-frontend/src/explore/components/ExploreChartPanel.test.jsx

```diff
@@ -145,4 +146,15 @@ describe('ChartContainer', () => {
     render(<ChartContainer {...props} />, { useRedux: true });
     expect(screen.queryByText('Cached')).not.toBeInTheDocument();
   });
+
+  it('hide gutter when collapse panel', async () => {
+    const props = createProps();
+    setItem(LocalStorageKeys.is_datapanel_open, true);
+    const { container } = render(<ChartContainer {...props} />, {
+      useRedux: true,
+    });
+    userEvent.click(screen.getByLabelText('Collapse data panel'));
+    const gutter = container.querySelector('.gutter');
```

Comment:
Can you please also verify in the test that this was visible before closing the data panel?